services/modules/slack/manager.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import unicode_literals
+import random
+import string
+import requests
+from eveonline.managers import EveManager
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from slackclient import SlackClient
+
+from six import iteritems
+from .tasks import SlackTasks
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlackManager:
+    def __init__(self):
+        pass
+
+    @classmethod
+    def _response_ok(cls, response):
+        try:
+            if response['ok']:
```

Comment:
You can turn this into a one line function, `return 'ok' in response and response['ok']`.